tardis/model/base.py

```diff
@@ -348,11 +357,16 @@ def from_config(cls, config):
             logger.warning("Abundances have not been normalized to 1."
                            " - normalizing")
             abundance /= norm_factor
-
+        
+        if hasattr(config.model, 'isotope_abundance'):
+            isotope_abundance = config.model.isotope_abundance
```

Comment:
no that's wrong - we have not decided on a config option and that should also come later!